core/client/src/main/java/alluxio/client/FileSystemTestUtils.java

```diff
@@ -37,10 +37,11 @@
    * @param fileName the name of the file to be created
    * @param len file size in bytes
    * @param options options to create the file with
-   * @throws IOException if {@code path} is invalid (e.g., illegal URI)
+   * @throws IOException if an IO exception occurs
+   * @throws AlluxioException if an Alluxio exception occurs
    */
   public static void createByteFile(FileSystem fs, String fileName, int len,
-      CreateFileOptions options) throws IOException {
+      CreateFileOptions options) throws IOException, AlluxioException {
```

Comment:
We use `IOException, AlluxioException` in 204 places and `AlluxioException, IOException` in 98 places >.< I'd prefer to punt alphabetization to a different PR